### PR TITLE
add a bew pa

### PR DESCRIPTION
**Pull request by johnmorrisQADeveloper**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <h1>My page</h1>
+  <p>pr is cool</p>
```
Comment (TechLover2020) [inline index.html:10]: please change this line

**Review by TechLover2020 — COMMENTED** [review]
Please address the review

File: index.html (hunk 1 above)
Comment (johnmorrisQADeveloper) [reply index.html:10]: tyhis is good

File: index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <h1>My page</h1>
```
Comment (TechLover2020) [inline index.html]: can you change this then please

File: index.html (hunk 1 above)
Comment (TechLover2020) [reply index.html:10]: okay

**Review by TechLover2020 — CHANGES_REQUESTED** [review]
🌵

**Review by TechLover2020 — APPROVED** [review]

**Pull request merged**